README.md

```diff
@@ -42,31 +42,73 @@ jobs:
           token: ${{ secrets.BUF_TOKEN }}
 ```
 
-See [action.yml](action.yml) for all options.
-
 ### Default behavior
 
-The default behavior of this action is the recommended workflow for a GitHub repository that contains Protobuf files.
+When you push a Git commit, tag, or branch to GitHub, the action will [push named modules to the BSR](https://buf.build/docs/bsr/module/publish) using `buf push`.
+
+On a pull request, the action will run all checks and post a [summary comment](#summary-comment) on the PR using `buf build`, `buf lint`, `buf format`, and `buf breaking`.
 
-| GitHub action event | Default behavior | `buf` commands |
-| - | - | - |
-| [`push`][push-event] | Modules that are configured with a BSR name are [pushed to the BSR](https://buf.build/docs/bsr/module/publish) every time a new Git commit, tag, or branch is pushed to GitHub. | `buf push` |
-| [`pull_request`][pull-request-event] | Run all checks and post (or update) a [summary comment](#summary-comment) on the PR every time the PR is updated. Errors are added as annotations on the PR. | `buf build`<br>`buf lint`<br>`buf format`<br>`buf breaking` |
-| [`delete`][delete-event] | Archive the corresponding label on the BSR every time a Git branch or tag is deleted from GitHub. | `buf beta registry archive --label` |
+When you delete a Git branch or tag, the action will archive the corresponding label on the BSR.
 
-### Skipping steps
+This behavior can be configured to fit your desired workflow.
 
-The default configuration makes it possible to skip lint, formatting, or breaking change checks on a PR
-by adding a label with a (case-insensitive) special name to that PR.
+### Configuration
 
-- `buf skip breaking`: skips breaking change detection.
-- `buf skip lint`: skips lint.
-- `buf skip format`: skips format. 
+To customize the behavior of the action, you can set the following inputs in the workflow file.
+Add these inputs under the `with` section of the `uses` step in the workflow file.
```

Comment:
If we are going to call it "parameter" in the table (and I think we should because otherwise it is confusing with the parameter we have that is called "input") then we need to call it "parameter" everywhere in this readme. This is just one occurrence, please check the full readme for necessary renames.

```suggestion
To customize the behavior of the action, you can set the following parameters in the workflow file.
Add these parameters under the `with` section of the `uses` step in the workflow file.
```

Reply:
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs
They use the term input parameters, so parameters is correct here.